### DISCUSSION TITLE
How to access the currently running DagsterInstance via python

### DISCUSSION THREAD

**Discussion by dpeng817** [dagster-io/dagster#12808] [Q&A] [answered]
The currently running instance (that is, the one configured in the `$DAGSTER_HOME` directory) can be accessed via a call to `DagsterInstance.get()`.

**Comment by dpeng817** [comment, marked as answer]
Answer is in the description

**Comment by sryza** [comment]
I believe this is a duplicate of https://github.com/dagster-io/dagster/discussions/7427
